From: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
Subject: Allow for very large packet/bytes transfered values
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/isc-dhcp/+bug/1222377

Allow for very large values in packet count or bytes transferred; thanks to
Paul (i41bktob) for identifying the issue.
---
 common/discover.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/common/discover.c
===================================================================
--- a/common/discover.c
+++ b/common/discover.c
@@ -417,7 +417,7 @@ struct iface_info {
  */
 int 
 begin_iface_scan(struct iface_conf_list *ifaces) {
-	char buf[256];
+	char buf[512];
 	int len;
 	int i;
 
